src/tests/App.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import App from "../App";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import userEvent from "@testing-library/user-event";
+
+it("should render Current Notifications", () => {
+  render(<App />);
+  const message = screen.queryByText(/Current Notification/i);
+  expect(message).toBeVisible();
+});
+
+it("renders the pagination bar", () => {
+  render(<App />);
+  const pageButton = screen.getByText("1");
```

Comment:
Is there a reference like a class name to the pagination bar that would be better to check for instead of getting the number 1 by text?

Reply:
no class name per se, in tailwind the styling is done through the class names and because with this hard coded data set, there's only 1 page, when we get an endpoint to pass in data, then i can mock that data with jest and test for multiple pages

Reply:
In addition, `React-Testing-Library` discourages any query that isn't visually based as their recommended approach is to test how the user experiences the app such as whatever text is displayed - in this case the `1`. There are cases where sometimes it does not make sense and if so, better to add a `data-testid` prop to the component which can then be queried.

Reply:
Ok got it, thanks for clarifying!